geopandas/array.py

```diff
@@ -280,6 +281,16 @@ def __init__(self, data, crs=None):
 
         self._crs = None
         self.crs = crs
+        self._sindex = None
+
+    @property
+    def sindex(self):
```

Comment:
Multiple at the same time? I was thinking this would work similar to how crs works where you set it in `__init__`. Are you suggesting it could work like `...sindex(type="rtree")` or something?

Reply:
> Are you suggesting it could work like ...sindex(type="rtree") or something?


That's a *possible* idea, yes, but something we should discuss at some point. 

But OK, let's leave it as is now in this PR. We can still convert it to a private attribute later before the next release if we think that's better for now (and the array is a bit hidden for the user anyway)